src/SimpleExcelReader.php

```diff
@@ -178,6 +182,21 @@ public function getRows(): LazyCollection
         });
     }
 
+    private function rewindRowIteratorToData(): void
+    {
+        $lastKey = null;
+        $this->rowIterator->rewind();
+
+        while (
+            null !== $this->headerRowKey
```

Comment:
It seems to me that looping through the entire file will hurt performance, especially for big files. Isn't there a method on the base package we can use for counting instead of implementing it ourselves?